File: app/src/main/java/com/github/overpass/gather/screen/meeting/enrolled/EnrolledViewModel.java
package com.github.overpass.gather.screen.meeting.enrolled;


import androidx.lifecycle.ViewModel;

public class EnrolledViewModel extends ViewModel {

    private boolean shouldPlayAnim = true;

    public boolean shouldAnimate() {
        if (shouldPlayAnim) {
            shouldPlayAnim = false;
            return true;
        } else {
            return false;
        }
    }
}

File: app/src/main/java/com/github/overpass/gather/screen/meeting/enrolled/EnrolledViewModel.kt
package com.github.overpass.gather.screen.meeting.enrolled


import androidx.lifecycle.LiveData
import androidx.lifecycle.ViewModel
import com.github.overpass.gather.model.commons.SingleLiveEvent
import javax.inject.Inject

class EnrolledViewModel @Inject constructor(
        private val playAnimationData: SingleLiveEvent<Void>
) : ViewModel() {

    init {
        playAnimationData.call()
    }

    fun playAnimation(): LiveData<Void> = playAnimationData
}
